<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureListItemWidget</class>
 <widget class="QWidget" name="TextureListItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QStackedWidget {
border: 1px solid rgb(95, 95, 95);
border-radius: 2px;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="addPage">
      <widget class="QPushButton" name="addPushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>96</width>
         <height>96</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;
border-radius: 0px;</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="texturePage">
      <widget class="QLabel" name="textureNameLabel">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>78</y>
         <width>95</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Texture name</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>6</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(218, 218, 218);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border: none;
     border-radius: 4px;
	background-color: rgba(0, 0, 0, 155);
min-width:10;
min-height: 10;
 }

 QPushButton:pressed {    
	background-color: rgba(80, 80, 80, 155);
padding-top: 0px;
padding-left: 0px;
 }</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
